data/json/items/tool_armor.json

```
@@ -3173,7 +3173,7 @@
     "type": "TOOL_ARMOR",
     "category": "armor",
     "name": { "str": "riot helmet" },
-    "description": "A helmet with a plastic shield that covers your entire face.  Activate to raise the face shield.",
+    "description": "A riot helmet with a plastic face shield that covers your entire face. Activate to raise the face shield.",
```

```suggestion
    "description": "A riot helmet with a plastic face shield that covers your entire face.  Activate to raise the face shield.",
```

Do not remove double space after periods. This is important for transifex.